VKApp/VKApp/Assets.xcassets/LoginWebViewController.swift

```diff
@@ -0,0 +1,91 @@
+// LoginWebViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+import WebKit
+
+/// Экран авторизациии в браузере
+final class LoginWebViewController: UIViewController {
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let shemeUrlComponent = "https"
+        static let hostUrlComponent = "oauth.vk.com"
+        static let pathUrlComponent = "/authorize"
+        static let loginWebGegieIdentifier = "LoginWebSegue"
+        static let queryItems = [
```

Comment:
```suggestion
        static let = [
``` 
в константы только текстовки, сам массив не надо